R/demandRegression.R

```diff
@@ -146,16 +145,21 @@ toolDemandReg <- function(tech_output, price_baseline, GDP_POP,
                   measure.vars = c("trn_aviation_intl", "trn_freight", "trn_pass", "trn_shipping_intl"))
   D_star = D_star[,.(region, year, demand = value, sector = variable)]
 
-  if (!is.null(demscen_factors)) {
```

Comment:
So `demscen_factors` are not allowed anymore to be NULL? This is not optimal since this is the default argument. If you want to force the user to provide `demscen_factors`, please remove the default argument.

Reply:
The default scenario is now "default"

Reply:
This leads to zero entries in the demScen tab which is checked in the demandRegression. I did not implement the option NULL as our techScens cannot be NULL either. But what indeed needs to be changed is the entry for mrremind with the default setting

Reply:
I see. Still, if the default argument is NULL, the function should be able to cope with NULL. I suggest to remove the default argument to `toolDemandReg` for `demscen_factors`.